lab-michael/route/recipes-routes.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const storage = require('../lib/storage.js');
+const Recipe = require('../model/recipe.js');
+
+module.exports = function(router){
+
+  router.get('/api/recipe', function(req, res) {
+    storage.fetchAll('recipe', req.params.id)
```

Comment:
If you make a request to `api/recipe`, how are you passing an ID into the `fetchAll` method? The intent of this route is that you can omit the ID and get all resources back.